rhg_compute_tools/kubernetes.py

```diff
@@ -149,12 +149,12 @@ def get_cluster(
 
     if cred_path is not None:
         container['env'].append({
-            'name': 'GCLOUD_DEFAULT_TOKEN_FILE',
+            'name': 'GOOGLE_APPLICATION_CREDENTIALS',
             'value': cred_path})
 
     elif cred_name is not None:
         container['env'].append({
-            'name': 'GCLOUD_DEFAULT_TOKEN_FILE',
+            'name': 'GOOGLE_APPLICATION_CREDENTIALS',
```

Comment:
This authenticates with our tools via prepare.sh in the worker image:
https://github.com/RhodiumGroup/docker_images/blob/master/worker/prepare.sh#L48

I'm pretty sure this change will break the process we use to authenticate gsutil for uploads. Not sure what is going on with the rm auth but I'm not sure this fixes it.

Reply:
this does fix it and I think will work with gsutil as well? https://cloud.google.com/docs/authentication/getting-started

Reply:
and I think we could then remove the associated lines in prepare.sh. But if you want to keep it, we can just make sure we explicitly point to the credentials path when calling `rhg_compute_tools.gcs.rm`. For whatever reason, just instantiating `google.cloud.Client()` without credentials does recognize the `GOOGLE_APPLICATION_CREDENTIALS` but does not recognize that we've authenticated them with `gcloud auth`

Reply:
@delgadom ok so it does break gsutil. I think we might as well use this for the env variable name, and just change prepare.sh to use 'GOOGLE_APPLICATION_CREDENTIALS' instead of 'GCLOUD_DEFAULT_TOKEN_FILE'. But for now, I think we'll just put in both env variables?